wireguard-install.sh

```diff
@@ -353,9 +387,9 @@ function uninstallWg() {
 		systemctl disable "wg-quick@${SERVER_WG_NIC}"
 
 		if [[ ${OS} == 'ubuntu' ]]; then
-			apt-get autoremove --purge -y wireguard qrencode
+			apt-get autoremove --purge -y wireguard wireguard-tools qrencode
 		elif [[ ${OS} == 'debian' ]]; then
-			apt-get autoremove --purge -y wireguard qrencode
+			apt-get autoremove --purge -y wireguard wireguard-tools qrencode
```

Comment:
Added wireguard-tools to the uninstall, as it is installed either way, but not purged if a container.